docs/source/intervention_models/iv_iron_antenatal/index.rst

```diff
@@ -83,77 +76,62 @@ Vivarium Modeling Strategy
     - Affected measure
     - Effect size measure
     - Effect size
-    - Note
-  * - 
-    - 
-    - 
-    - 
-    - 
-    - 
+    - Note 
+  * - Hemoglobin
+    - Modelable entity
+    - 10487
+    - Population mean hemoglobin concentration (as continuous measure)
+    - Mean difference
+    - +10 g/L
     - 
+  * - Birthweight
+    - Risk exposure
+    - 339
+    - Population mean birthweight (as continuous measure)
+    - Mean difference
+    - +50 grams
+    - Assume no difference in gestational age
 
 Hemoglobin level among pregnant and lactating women
 +++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-.. todo::
-
-  Link to existing document of the affected outcome (ex: cause or risk exposure model document)
-
-.. todo::
-
-  Describe exactly what measure the intervention will affect
-
-.. todo::
-
-  Fill out the tables below
-
-.. list-table:: Maternal hemoglobin restrictions
-  :widths: 15 15 15
-  :header-rows: 1
-
-  * - Restriction
-    - Value
-    - Note
-  * - Male only
-    - 
-    - 
-  * - Female only
-    - 
-    - 
-  * - Age group start
-    - 
-    - 
-  * - Age group end
-    - 
-    - 
-  * - Other
-    - 
-    - 
-
 .. list-table:: Maternal hemoglobin effect size
   :widths: 15 15 15 
   :header-rows: 1
 
   * - Population
     - Effect size
     - Note
-  * - 
-    - 
-    - 
-.. todo::
+  * - Pregnant women with hemoglobin levels less than 100 g/L
+    - +10 g/L 
+    - Effect is present two weeks after start of pregnancy through the end of the postpartum period. Effect size is from the BMGF optomistic target product profile relative to oral iron supplementation
 
-  Describe exactly *how* to apply the effect sizes to the affected measures documented above
+The effect of the IV iron intervention on maternal hemoglobin should be applied as an additive shift to the simulant's continuous hemoglobin exposure value two weeks after the start of pregnancy after the application of the pregnancy adjustment factor. The effect of the intervention should be removed (subtracted from the hemoglobin exposure level) at the end of the postpartum period prior to the removal of the pregnancy adjustment factor.
 
-.. todo::
+.. note::
 
-  Note research considerations related to generalizability of the effect sizes listed above as well as the strength of the causal criteria, as discussed on the :ref:`general research consideration document <general_research>`.
+  The impact of antenatal IV iron should persist despite other changes to the materna hemoglobin level due to the maternal supplementation intervention, maternal hemorrhage incidence, or postpartum IV iron.
 
 Assumptions and Limitations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- To align with the optomistic target profile from the BMGF, we assume that IV iron is administered at the very start of pregnancy. This will overestimate the impact of the intervention given this is unrealistically early for most women.
+
+.. todo::
```

Comment:
Consider including measurement error in the measurement of Hb level.  Catherine has a paper with measurement error for many different labs that could inform this.